tex/main/MarkovChain/closed_form_state_probs/main.tex

```diff
@@ -368,60 +379,61 @@ \subsubsection{Conjecture of adding columns (combinatorics attempt for closed fo
 \end{equation}
 
 Note here that the final value \(U_{p_1,p_2,p_3}\) corresponds to coefficients that are unknown and are currently investigated. 
-In essence function \(G()\) takes as arguments a possible combination of numbers of \(\lambda^A, \lambda^o \text{ and } \mu \) for a given Markov model and outputs the coefficient of that term which in turn represents how many spanning trees exist in the graph with that specific combination. 
+The function \(R()\) takes as arguments a possible combination of numbers of \(\lambda^A, \lambda^o \text{ and } \mu \) for a given system and outputs the coefficient of that term which in turn represents how many spanning trees exist in the graph with that specific combination. 
```

Comment:
You're missing things from my suggestion (smaller diffs would make this less likely to happen).

```suggestion
The function \(R\) takes as arguments a possible combination of numbers of \(\lambda^A, \lambda^o \text{ and } 
\mu \) for a given system and outputs the coefficient of that term which in turn represents how many spanning
 trees exist in the graph with that specific combination. 
```

Reply:
Why the change from `G` to `R`?

Reply:
I was thinking about this over breakfast and it's because we're using \(G\) to refer to the graph right? (Make sense :))

Reply:
Yeah was just about to reply. That's exactly why. G_M is the graph associated with the Markov chain

Reply:
Cool. Makes sense (but yeah things like these, document them in the commit message body or a comment on the PR :)).